bootloaders/uart/uart_binary.c

```diff
@@ -49,9 +70,9 @@ void qspi_gpio_set_function(uint gpio, gpio_function_t fn) {
 int main() {
     pico_led_init();
 
-    // SD2 is QSPI GPIO 3, SD3 is QSPI GPIO 4
-    qspi_gpio_set_function(3, GPIO_FUNC_UART_AUX);
-    qspi_gpio_set_function(4, GPIO_FUNC_UART_AUX);
+    // SD2 is UART0 TX, SD3 is UART0 RX (same as the ROM UART bootloader)
+    qspi_gpio_set_function(QSPI_GPIO_SD2, GPIO_FUNC_UART_AUX);
+    qspi_gpio_set_function(QSPI_GPIO_SD3, GPIO_FUNC_UART_AUX);
```

Comment:
While here, these should be `GPIO_FUNC1_UART_AUX` (it's the same number, but this is the define from the io_qspi.h header)

```suggestion
    qspi_gpio_set_function(QSPI_GPIO_SD2, GPIO_FUNC1_UART_AUX);
    qspi_gpio_set_function(QSPI_GPIO_SD3, GPIO_FUNC1_UART_AUX);
```

Reply:
agreed on all. I pushed a fix.